components/brave_shields/adblock/rs/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "adblock-cxx"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+cxx = "1.0"
+adblock = { git = "https://github.com/djandries/adblock-rust", default-features = false, features = ["full-regex-handling", "object-pooling", "unsync-regex-caching", "debug-info", "css-validation"] }
```

Comment:
Will switch out once brave/adblock-rust#275 is resolved

Reply:
ok noted, i think i was just copying over existing features from the original ffi cargo manifest

Reply:
@antonok-edm could you please help me out with a version bump? thanks https://github.com/brave/adblock-rust/pull/280

Reply:
done ^